open-machine-learning-jupyter-book/deep-learning/cnn/cnn.ipynb

```diff
@@ -2,13 +2,25 @@
  "cells": [
```

Comment:
<p>There's a lot of WARNING, try to fix it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>